src/app/notifications/page.tsx

```
@@ -40,46 +47,46 @@ const TAB_OPTIONS = {
-};
+  )
+}
 
 const FormattedUsersText = ({ users }: { users: User[] }) => {
   if (users.length === 0) {
-    return "Nobody";
+    return "Nobody"
   }
 
   const firstUserLink = (
     <PriorityLink href={`/s/${users[0].username}`} className="hover:underline">
```

ESLint

[error] 'PriorityLink' is not defined.

(react/jsx-no-undef)